workers/data_refinery_workers/processors/smashing_utils.py

```diff
@@ -375,9 +385,13 @@ def process_frames_for_key(key: str,
 
         if frame_data is None:
             job_context['unsmashable_files'].append(computed_file.filename)
+            job_context['filtered_samples'][sample.accession_code] = {
+                'reason': 'We were unable to smash the file associated with this sample during the second pass.',
```

Comment:
Once again, they won't know what second pass means. Perhaps `The file associated with this sample did not contain a vector that fit the expected dimensions of the matrix.`? I'm less sure about this one. I honestly wouldn't think we'd see many of these.